### PR TITLE
Update i18n topic - master

### LINKED ISSUE #2580: The localization topic is in a wrong place, should include i18n in the title and is outdated.
Multiple issues here:

1. The localization topic should be in the general section, not in the services section. 
2. The title should include i18n, because that's what people search by [e.g. Localization (i18n)]. 
3. The sample doesn't actually show multiple languages, it shows a mechanism for changing a filtering chip label with another label again in EN. The sample should show multiple languages + language not supported by the `igniteui-angular-i18n` package (e.g. BG).
4. The languages come on the bottom of the topic and the list is outdated. The topic should lead with the supported languages out of the box.

### PR DESCRIPTION
**Pull request by IvayloG**
Closes #2580  

### Checklist:
 
 - [x] check topic's TOC/menu and paragraph headings
 - [ ] link to other topics using `../relative/path.md`
 - [ ] at the References section at the end of the topic add links to topics, samples, etc
 - [ ] reference API documentation instead of adding a section with API

<br />

 - [ ] use valid component names - [Data] Grid, `IgxSelectComponent`, `<igx-combo>`
 - [x] use spell checker tool (VS Code, Grammarly, Microsoft Editor)
 - [x] add inline `code blocks` for the names of classes / tags / properties
 - [x] add language descriptor for the ```code blocks```
 - [ ] check broken links (use browser add-on)
 - [x] check if sample is working and fully visible in the topic
 - [x] check if sample is working and fully visible in the StackBlitz
 - [x] check if code blocks match the code in StackBlitz demo

<br />

 - [x] follow SEO guidelines to fill topic's metadata ([SEO guidelines](https://infragisticsinc297.sharepoint.com/:w:/g/Groups/marketing/EWz9InT4FDlErHCumxsKGY4Bd8H03yhRWxDFk47luRz-_Q?e=S5wWcx))

<br />

 - [ ] do not resolve requested changes (leave that to the reviewer)
 - [ ] add `pending-localization` label when the review of the PR is done
 - [ ] add a member from the localization team to translate it

### CODE REVIEW

**Review by Lipata — DISMISSED** [review]

File: en/components/general/localization.md [hunk 1]
```diff
@@ -1,32 +1,87 @@
...
-title: Localization - Native Angular | Ignite UI for Angular
+title: Localization (i18n) - Native Angular | Ignite UI for Angular
 _description: The Ignite UI for Angular Localization mechanism provides the ability to change/localize strings in the components.
 _keywords: Ignite UI for Angular, UI controls, Angular widgets, web widgets, UI widgets, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components Library, Native Angular Components
 ---
 
-# Localization
+# Localization (i18n)
+
+Currently, Ignite UI for Angular ships with resource strings for the following languages: German, French, Italian, Japanese, Korean, Spanish, Traditional Chinese (zh-Hant) and Simplified Chinese (zh-Hans). These are available via the `igniteui-angular-i18n` package.
```
Comment (valadzhov) [inline en/components/general/localization.md:9]: ...for the following languages **and scripts**
...**traditional and simplified Chinese** (I think there is no need for the zh-Hant and zh-Hans).

File: en/components/general/localization.md [hunk 2]
```diff
@@ -1,32 +1,87 @@
...
+<button data-localize="codesandbox" disabled class="codesandbox-btn" data-iframe-id="localization-all-resources-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on codesandbox</button>
+<button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="localization-all-resources-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
 </div>
 
 ## Usage
 
-### Localize entire application
+### Load localized resources from npm package
+
+To localize an application for one of the available in the `igniteui-angular-i18n` package languages, the most straight forward approach is to:
```
Comment (valadzhov) [inline en/components/general/localization.md:27]: You can localize an application in one of the languages available in the `igniteui-angular-i18n` package in the following way:

File: en/components/general/localization.md [hunk 3]
```diff
@@ -1,32 +1,87 @@
...
+
+Install the package first:
```
Comment (valadzhov) [inline en/components/general/localization.md:29]: 1. Install the package by executing `npm install igniteui-angular-i18n --save-dev`

File: en/components/general/localization.md [hunk 4]
```diff
@@ -1,32 +1,87 @@
...
+
+`npm install igniteui-angular-i18n --save-dev`
+
+Then import the resource strings for the desired language and call the `changei18n` function passing the corresponding resource object:
```
Comment (valadzhov) [inline en/components/general/localization.md:33]: 2. Import the resource strings for the desired language and call the `changei18n()` function and pass the corresponding resource object.

File: en/components/general/localization.md [hunk 5]
```diff
@@ -1,32 +1,87 @@
...
+import { IgxResourceStringsJA } from 'igniteui-angular-i18n';
+...
+public ngOnInit(): void {
+    ...
+    changei18n(IgxResourceStringsJA);
+    ...
+}
+```
+
+>Note: Feel free to contribute to the `igniteui-angular-i18n` package.
```
Comment (valadzhov) [inline en/components/general/localization.md:48]: Feel free to submit any ideas or changes related to the `igniteui-angular-i18n` package by logging them in the [repository](https://github.com/IgniteUI/igniteui-angular-i18n/)

File: en/components/general/localization.md [hunk 6]
```diff
@@ -1,32 +1,87 @@
...
+
+### Utilize own localized resources
```
Comment (valadzhov) [inline en/components/general/localization.md:50]: ### Utilize custom localized resources

**Review by Lipata — APPROVED** [review]

**Pull request merged**